README.md

```diff
@@ -53,6 +53,15 @@ docker run -it -p 8888:8888 \
 
 > Note: New files are created inside Docker and will persist unless --rm option is used. If you'd like to also save the credentials and xRunBooks on your local machine, use Docker's -v parameter when you start the container.
 
+### Kubernetes
+The Docker image can be installed in K8s:
+
+* Deploy using the following command ([the Yaml file](installation/unskript-oss-k8s-deployment.yaml))
```

Comment:
helm chart is the standard way of deploying, we should instead build a helm chart for deploying the docker.

Reply:
Should this task be assigned to @jayasimha-raghavan-unskript ?

What about this PR? Should we merge it or delete it @amit-chandak-unskript ?

Reply:
Yes should be assigned to @jayasimha-raghavan-unskript . We can close this PR